lua/ayu/init.lua

```diff
@@ -276,6 +276,9 @@ local function set_groups()
     VM_Cursor = { bg = colors.selection_inactive, sp = colors.fg, underline = true },
     VM_Insert = { sp = colors.fg, underline = true },
     VM_Mono = { fg = colors.bg, bg = colors.comment },
+
+    -- TreesitterContext.
+    TreesitterContext = { bg = colors.selection_inactive },
```

Comment:
One more request. Could you put it after the `TreeSitter` block above?

Reply:
Done!